_posts/2023-06-26-trino-fest-2023-salesforce.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Trino Fest 2023 recap: Anomaly detection for Salesforce’s production data using Trino"
+author: "Tuli Nivas, Geeta Shankar, and Ryan Duan"
```

Comment:
Please make sure we let them know about the PR in an email today @colebow and @ryandoowan  .. and give them the heads up that we will publish on Monday and also socialize then